notebooks/02-ingestion/02-batch-ingestion.ipynb

```diff
@@ -976,7 +1065,8 @@
     "druid.datasources.drop('kttm-nested-batch', True)\n",
     "druid.datasources.drop('kttm-enhanced-batch', True)\n",
     "druid.datasources.drop('wiki-users-batch', True)\n",
```

Comment:
These table names don't conform to the standard - e.g. should be example-wikipedia-{etc}

Reply:
yeah, this was an earlier notebook. I'll change them all.